packages/faustwp-core/src/hooks/useLogout.tsx

```diff
@@ -25,8 +25,17 @@ export function useLogout() {
       return;
     }
 
+    const isPreview = window?.location?.search?.includes('preview=true');
+
     if (redirectUrl) {
       window.location.assign(redirectUrl);
+    } else if (isPreview) {
+      const publicUrlPath = process.env.NEXT_PUBLIC_URL;
+      if (publicUrlPath) {
+        window.location.assign(publicUrlPath);
+      } else {
+        window.location.reload();
```

Comment:
When `NEXT_PUBLIC_URL` is not defined, logging out results in the preview page reloading and logging me back in.

Reply:
https://github.com/wpengine/faustjs/assets/1254870/31c40aab-4064-4f2c-962f-ec46ed282eec

Reply:
Updated to go to '/'